samples/wifi/radio_test/src/nrf_wifi_radio_test_shell.c

```diff
@@ -989,10 +989,10 @@ static int nrf_wifi_radio_test_set_rx_capture_length(const struct shell *shell,
 
 	val = strtoul(argv[1], &ptr, 10);
 
-	if (val >= 16384) {
+	if ((val > 16383)  || (val < 0)) {
```

Comment:
Generic comment: What is the source of the magic numbers used throughout in the file ? If these are limitations arising from the FW then please add corresponding defines in the relevant FW shared header file and then use the defines in this file.